cli/tests/test_completion.rs

```diff
@@ -404,6 +404,99 @@ fn test_completions_are_generated() {
     ");
 }
 
+#[test_case(Shell::Bash; "bash")]
+#[test_case(Shell::Zsh; "zsh")]
+#[test_case(Shell::Fish; "fish")]
+fn test_default_command_is_resolved(shell: Shell) {
+    let test_env = TestEnvironment::default();
+
+    let output = test_env
+        .complete_at(shell, 1, ["--"])
+        .take_stdout_n_lines(2);
+    match shell {
+        Shell::Bash => {
+            insta::assert_snapshot!(output, @r"
+            --revisions
+            --limit
+            [EOF]
+            ");
+        }
+        Shell::Zsh => {
```

Comment:
Maybe these tests were snapshotted after the fix? To demonstrate the bug, you'll need to run tests at this revision. You can also squash the added tests to the last patch.

Reply:
I created the snapshots originally by "manually" resolving the aliases. The tests were intentionally failing in those two revisions. Personally, I don't mind that as it is then easier to prove that the tests are actually sensitive to the bug that's fixed in a later commit. But I also understand the desire to have each revision pass CI individually. I've squashed them, as you've suggested.